src/lib/db.js

```diff
@@ -40,9 +40,9 @@ export const AppPages = {
 export const PageToRoute = {
   [AppPages.OPTIMIZER]: BASE_PATH,
 
-  [AppPages.RELIC_SCORER]: BASE_PATH + '#scorer',
-  [AppPages.CHANGELOG]: BASE_PATH + '#changelog',
-  [AppPages.GETTING_STARTED]: BASE_PATH + '#getting-started',
+  [AppPages.RELIC_SCORER]: BASE_PATH + '/scorer',
+  [AppPages.CHANGELOG]: BASE_PATH + '/changelog',
+  [AppPages.GETTING_STARTED]: BASE_PATH + '/getting-started',
```

Comment:
This is actually not a valid change on github pages even though it looks like it works locally

Reply:
https://github.com/orgs/community/discussions/64096